spec/spec_helper.rb

```diff
@@ -152,7 +152,6 @@
 
 # Capybara.javascript_driver = :webkit
 # Capybara.default_driver = :sniffybara
-Capybara.default_driver = :selenium_chrome
-Capybara.enable_aria_label = true
-
-# Capybara.javascript_driver = :chrome
+# Capybara.default_driver = :selenium_chrome_headless # Uncomment to debug feature tests 
```

Comment:
Should this just be `selenium_chrome`? It looks like the same code as line `156`.

Reply:
😥 aw dangit... I'll fix that